modules/ROOT/pages/indexes-for-search-performance.adoc

```diff
@@ -662,6 +663,21 @@ This command will produce a table with the following columns:
 | The name of the constraint the index is associated with or `null` if the index is not associated with any constraint. label:default-output[]
 | `STRING`
 
+// New in 5.8
+| `lastRead`
+| The last time the index was used for reading, or `null` if it has not yet been read since `trackedSince` or the statistics are not tracked. label:default-output[]
```

Comment:
```suggestion
| The last time the index was used for reading. `null` if it has not yet been read since `trackedSince` or the statistics are not tracked. label:default-output[]
```
Otherwise it's a little ambiguous which `or` gets "evaluated" first.